tools/cross-toolchain/regenerate.sh

```diff
@@ -9,4 +9,4 @@ cp -vf "${RBE_AUTOCONF_ROOT}/tools/cross-toolchain/empty.bzl" "${RBE_AUTOCONF_RO
 
 # Bazel query is the right command so bazel won't fail itself.
 bazel query "@rbe_ubuntu_clang_gen//..."
-bazel query "@rbe_ubuntu_gcc_gen//..."
\ No newline at end of file
+#bazel query "@rbe_ubuntu_gcc_gen//..."
```

Comment:
Do we need to keep this commented?

Reply:
No, we are using clang only at the moment.

Reply:
I'll remove it